core/src/main/java/org/ehcache/spi/cache/ObservableCacheManager.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ehcache.spi.cache;
+
+
+import org.ehcache.core.events.CacheManagerListener;
+
+/**
+ * Specifies methods for a cache manager observable by the management framework.
+ *
+ * @author Clifford W. Johnson
+ */
+public interface ObservableCacheManager {
```

Comment:
Anyone else want to weight in on this?  @chrisdennis @ljacomet ?  I have no objection to making this extend `CacheManager` ...